internal/database/database.go

```diff
@@ -108,3 +125,40 @@ func (db *dbbase) withTx(ctx context.Context, fn func(tx *sqlx.Tx) error) error
 
 	return err
 }
+
+// get a string and return errors if it is not in json format
+func IsJson(input string) error {
+	var jsonData interface{}
+	err := json.Unmarshal([]byte(input), &jsonData)
+	if err != nil {
+		return err
+	} else {
+		return nil
+	}
+}
+
+func Jsonify(uc model.UserConfig) ([]byte, error) {
```

Comment:
A simple `json.Marshal` of `model.UserConfig` should be enough to serialize it. This logic belong outside of the database. Did you encounter any issues with this?

Reply:
I believe this function does not exist anymore, so I'm resolving this.